.travis.yml

```diff
@@ -32,6 +32,7 @@ before_deploy:
 jobs:
   include:
   - stage: deploy
+    if: branch = master
```

Comment:
I guess CI will confirm in a few mins but, if you want this to skip if the branch isn't master, is this the right equality?

Reply:
I think so.  I am following the example here: https://docs.travis-ci.com/user/build-stages/#Specifying-Stage-Order-and-Conditions

Reply:
Oops... looks like I put it in the wrong spot.  Will fix.